client/landing/stepper/declarative-flow/internals/hooks/use-launchpad-decider/index.tsx

```diff
@@ -22,42 +24,37 @@ interface SiteProps {
 }
 
 export const useLaunchpadDecider = ( { exitFlow, navigate }: Props ) => {
-	// placeholder field for the experiment assignment
-	const showCustomerHome = false;
-	const sessionId = getSessionId();
-	let launchpadStateOnSkip: null | 'skipped' = null;
-	if ( showCustomerHome ) {
-		launchpadStateOnSkip = 'skipped';
+	const { getIntent } = useSelect( ( select ) => select( ONBOARD_STORE ) as OnboardSelect, [] );
+	const intent = getIntent();
+	const site = useSite();
+	// site intent is not set until we exit site setup flow so we set it here with onboarding store
```

Comment:
It may be worth being a bit more specific with the comment in this case. We're not actually "setting the site intent" as I commonly understand it, because we're not setting the server-side option. We are splicing the value into the client-side data, and I think we should call that out.
```suggestion
	// The site_intent option is not set until we exit site setup flow so we set the client value with data from the onboarding store
```

Reply:
Yes that make sense so we are clear that we are not setting the intent from a server side perspective.